src/backend/InvenTree/plugin/samples/machines/sample_printer.py

```diff
@@ -44,7 +53,14 @@ def print_label(
         **kwargs,
     ) -> None:
         """Send the label to the printer."""
+        print_delay = int(machine.get_setting('DELAY', 'D', backup_value=0) or 0)
```

Comment:
Is that int cast and or 0 really necessary? I mean in theory it should also convert it to an int because that's the type of the setting?